src/Microsoft.AspNet.Authentication.Facebook/FacebookAuthenticationMiddleware.cs

```diff
@@ -37,11 +37,11 @@ public class FacebookAuthenticationMiddleware : OAuthAuthenticationMiddleware<Fa
         {
             if (string.IsNullOrWhiteSpace(Options.AppId))
             {
-                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options.AppId)));
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Resources.Exception_OptionMustBeProvided, nameof(Options)));
```

Comment:
The parameter to an ArgumentException is the name of the argument. Is there a better way to do this exception?